### DISCUSSION TITLE
Notebook Function

### DISCUSSION THREAD

**Discussion by mollymorsebelcher** [ManifoldScholar/manifold#3780] [Q&A]
For a project, we are looking to create the feel of a personal notebook. Is there any way for users to have their own personal notebook in their profile where they can write answers to questions from the curriculum they are using? Additionally, for creators, is there any way to make each curriculum personal to the user taking it so that users can write answer to questions within the curriculum text? 

In short, I'm looking for a function that allows students to write their own answers to the prompts in the project we are moving to Manifold.

**Comment by tsmyre** [comment]
Users have the ability to leave annotations either publicly or privately (depending on the instance’s settings), as well as within the context of reading groups that can likewise be public, private, or anonymous. Using the notebook drop down from within the Manifold reader, users can see where and in what context they've left annotations. And each reading group also has its own landing page where annotations from everyone within the group can be seen and explored.

As such, instructors have the opportunity to create a private reading group for their students and leave prompts for them to respond to.
